QuestionnaireResponse/QuestionnaireResponse-RoD-PheobeSmithamMother-Example.json

```diff
@@ -0,0 +1,240 @@
+      "value": "9999999996"
+    }
+  },
+  "source": {
+    "reference": "Patient/Patient-PheobeSmithamMother-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307246"
+    }
+  },
+  "item": [
+    {
+      "linkId": "patientDetails",
+      "text": "Patient Details",
+      "item": [
+        {
+          "linkId": "givenName",
+          "text": "First Name",
+          "answer": [
+            {
+              "valueString": "Alice"
+            }
+          ]
+        },
+        {
+          "linkId": "familyName",
+          "text": "Last Name",
+          "answer": [
+            {
+              "valueString": "Smitham"
+            }
+          ]
+        },
+        {
+          "linkId": "nhs_Number",
+          "text": "NHS number (or postcode if not not known)",
+          "answer": [
+            {
+              "valueString": "9449307246"
+            }
+          ]
+        },
+        {
+          "linkId": "birthDate",
+          "text": "Date of Birth",
+          "answer": [
+            {
+              "valueDate": "1983-03-22"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "declaration4",
+      "text": "Confirmation of Your Genomic Test and Research Choices",
+      "item": [
+        {
+          "linkId": "confirmation",
+          "text": "I confirm that I have had the opportunity to discuss information about genomic testing, I agree to the genomic test, and my research choice is indicated below.",
+          "item": [
+            {
+              "linkId": "researchConfirmation1",
+              "text": "I have discussed taking part in the National Genomic Research Library. If your answer to A is NO then please ignore B and sign directly below",
+              "answer": [
+                {
+                  "valueBoolean": true
+                }
+              ]
+            },
+            {
+              "linkId": "researchConfirmation2",
+              "text": "I agree that my data and remainder sample may contribute to the National Genomic Research Library",
+              "answer": [
+                {
+                  "valueBoolean": true
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "isRespondentAttorney",
+      "text": "Are you completing this form on behalf of someone?",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "linkId": "patientValidation",
+      "text": "Patient Validation",
+      "item": [
+        {
+          "linkId": "patientNamecombined",
+          "text": "Patient Name",
+          "answer": [
+            {
+              "valueString": "Alice Smitham"
+            }
+          ]
+        },
+        {
+          "linkId": "patientSignature",
+          "text": "Signature",
+          "answer": [
+            {
+              "valueString": "AliceSmitham"
```

Comment:
Check QuestionnaireResponse logic, this should not be needed for remote consent, not a blocker though

Reply:
Patient signature currently has constraint required. So, this will require a restructuring of NHSDigital-Questionnaire. Need to change the position of checking if consent was taken remotely and attach condition to require patient signature.
Thus, I will raise a new branch for this.